sdk/tables/Azure.Data.Tables/src/TableInternalClient.cs

```diff
@@ -2,12 +2,22 @@
 // Licensed under the MIT License.
 
 using Azure.Core;
+using Azure.Core.Pipeline;
 
 namespace Azure.Data.Tables
 {
     // https://github.com/Azure/autorest.csharp/issues/451 .
     [CodeGenClient("TableClient")]
     internal partial class TableInternalClient
     {
+        internal string version { get; }
+
+        internal TableInternalClient(ClientDiagnostics clientDiagnostics, HttpPipeline pipeline, string url, string version = "2019-02-02")
```

Comment:
This version string needs to be yanked into a constant somewhere.  [I try to keep all Version info in the ClientOptions](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/search/Azure.Search.Documents/src/SearchClientOptions.cs#L17) so it can all be updated in one file when you're moving to a newer one.  Storage was a little more complicated because I wanted to [consolidate the shared versioning info across services.](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Common/src/Shared/StorageVersionExtensions.cs)

Reply:
I believe this is the default created by the generator which is the service version from the swagger. There is a const [defined in the options](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/tables/Azure.Data.Tables/src/TableClientOptions.cs#L23) but I think this version should be getting passed by the default ctor here to ensure we respect the defined options value, which isn't happening now.

Reply:
Yep, that `ClientOptions.VersionString` value should be making its way over here at some point.

Reply:
created #11856 for this